tools/create_f2fs_image.sh

```diff
@@ -0,0 +1,51 @@
+#!/bin/bash
+SYZDIR=/usr/local/google/home/jiahengh/gopath/src/github.com/google/syzkaller
+lab1="-a 1 "
```

Comment:
We should also test "-a 0"

Reply:
It seems that (contrary to the documentation) the default value is 1. So not having -a 1 would be a = 0

Reply:
Right, then not having it would be equivalent to `-a 1`.  That's why we also want to test `-a 0`